ofl/gulzar/METADATA.pb

```diff
@@ -21,3 +21,4 @@ source {
   repository_url: "https://github.com/simoncozens/Gulzar"
   commit: "7f736710573840523e68ea977843f7af7239632b"
 }
+minisite_url: "https://gulzarfont.org"
```

Comment:
There is an open PR for Gulzar #5530, we could avoid conflict with adding the minisite there

Reply:
Since Gulzar is blocked for now, let's add the mini site here and deal with the conflict later. cc @simoncozens